modules/virt-access-configuration-considerations.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: REFERENCE
+[id="virt-access-configuration-considerations_{context}"]
+= Access configuration considerations
+
+Each method for configuring access to a virtual machine (VM) has advantages and limitations, depending on the traffic load and client requirements.
+
+Services provide excellent performance and are recommended for applications that are accessed from outside the cluster.
+
+If the internal cluster network cannot handle the traffic load, you can configure a secondary network.
+
+Virtctl SSH and port-forwarding::
+* Simple to configure.
+* `virtctl ssh` recommended for troubleshooting VMs.
+* `virtctl port-forwarding` recommended for automated configuration of VMs with Ansible.
+* Not recommended for high-traffic applications like Rsync or Remote Desktop Protocol because of the burden on the API server.
+* The API server must be able to handle the traffic load.
+* The clients must be able to access the API server.
+* The clients must have access credentials for the cluster.
+
+Cluster IP service::
+* The internal cluster network must be able to handle the traffic load.
+* The clients must be able to access an internal cluster IP address.
+
+Node port service::
+* The internal cluster network must be able to handle the traffic load.
+* The clients must be able to access at least one node.
+
+Load balancer service::
+* A load balancer must be configured.
+* Each node must be able to handle the traffic load of one or more load balancer services.
+
+Secondary network::
+* Excellent performance because traffic does not go through the internal cluster network.
+* Allows a flexible approach to network topology.
+* Guest operating system must be configured with appropriate security because the VM is exposed directly to the secondary network. If a VM is compromised, an intruder could gain access to the secondary network.
+
```

Comment:
@apinnick  should we link here to the "network" section in https://access.redhat.com/articles/6994974 ?
cc @jeniferh

Reply:
I looked at the network section in the "tuning" article and I am not sure which section you think is relevant. networkInterfaceMultiqueue? Multus?

Reply:
Yes, most relevant is Multus and SR-IOV

Reply:
The sections point back to our own documentation for creating Linux bridge and SR-IOV network. If you want the user to know where that documentation is, I can make sure that the links appear in the sections on using secondary networks. We cannot add an internal cross-reference in a module, although I could add external links to the KB. But I am not sure that this is the best place for the links.

Reply:
Here is the section on [using secondary networks](https://61419--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-accessing-vm-ssh.html#using-secondary-networks-ssh_virt-accessing-vm-ssh). I added links to the prerequisites section.